server/src/main/scala/com/ubirch/user/server/route/UserRoute.scala

```diff
@@ -379,6 +386,23 @@ class UserRoute(implicit mongo: MongoUtil, val system: ActorSystem) extends CORS
     }
   }
 
+  private def getUsersByIds(userIds: List[UUID]): Route = {
```

Comment:
would be nice, if we would query this before sending the recalculateTrigger to Kafka. To have a proper responds to the person sending the request.